### PR TITLE
timeout call notification

### PR DESCRIPTION
**Pull request by mahibi**
this will hangup:
- the call notification after 1 minute (the popup at the top)
- the call notification screen after 1 minute (the screen with the hangup and accept call buttons)

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]
LGTM

File: app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt [hunk 1]
```diff
@@ -337,29 +341,23 @@ class MagicFirebaseMessagingService : FirebaseMessagingService() {
                     hasParticipantsInCall = participantList.isNotEmpty()
                     if (hasParticipantsInCall) {
                         for (participant in participantList) {
-                            if (participant.userId == signatureVerification.userEntity.userId) {
+                            if (participant.actorId == signatureVerification.userEntity.userId) {
```
Comment (nickvergessen) [inline app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt:344]: also have to compare the actorType?

Comment (mahibi) [reply app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt]: added check for actorType

**Comment by mahibi** [comment]
> you could also get the single notification and check the text there. then you automatically have the updated text available when it changed

@nickvergessen i don't get it

**Comment by mahibi** [comment]
merged this PR to fix at least the invinite ringing. 
follow up issue is https://github.com/nextcloud/talk-android/issues/1724

**Comment by AndyScherzinger** [comment]
@mahibi backport to 13.0 failed btw.

**Comment by mahibi** [comment]
> @mahibi backport to 13.0 failed btw.
@AndyScherzinger 

yep will do.
but will backport it to 13.0.0 ...?
asking because you set it to 13.1.0..

**Comment by AndyScherzinger** [comment]
@mahibi yes, because this PR has targeted master which is already 13.1, so the backport PR will then be 13.0 😊

**Comment by timkrueger** [comment]
> @mahibi yes, because this PR has targeted master which is already 13.1, so the backport PR will then be 13.0 blush

But when we backport this PR to 13.0 then the milestone must also 13.0. Or what I misunderstand?

**Comment by AndyScherzinger** [comment]
> But when we backport this PR to 13.0 then the milestone must also 13.0. Or what I misunderstand?

Yes but only on the newly created one by the backport bot which will set it correctly by itself :)

**Pull request merged**